ui/src/components/Model.js

```diff
@@ -38,8 +55,9 @@ const fetchData = async ({ setState, modelName }) => {
   const { data } = await api({
     endpoint: `${globals.VERSION}/graphql`,
     body: {
-      query: `query($filters: JSON) {
+      query: `query ModelDataQuery($filters: JSON) {
```

Comment:
I think if you add `ModelDataQuery`  after `/graphql` in the endpoint, it will show up in the chrome network panel as that and the query will still work: 

```
endpoint: `${globals.VERSION}/graphql/ModelDataQuery`,
```

Reply:
or wait, maybe that was just in gdc. should add that here as well https://github.com/overture-stack/arranger/blob/master/modules/server/src/startProject.js#L224

Reply:
works here too :sparkles: w00t